frontend/src/components/object/ObjectTable.vue

```diff
@@ -54,6 +54,39 @@ const tableData: Ref<Array<COMSObjectDataSource>> = ref([]);
 // Actions
 const toast = useToast();
 
+// is "select all" checked?
+const selectAll = ref(false);
+
+// list of objects, in the order dislpayed in the DataTable
+const objectsList: Ref<Array<COMSObject>> = ref([]);
+
+// Initial page load?
+const isInitialLoad = ref(true);
+
+// DataTable pagination state
+const itemsPerPage = ref(10);
+const currentPageTop = ref(0);
```

Comment:
Do we have these magic numbers as constants somewhere? I know we generally use 10 as our default pagination size, but that should be defined as a constant across the codebase so that we're not just raw-defining default integer states.

Reply:
It was previously a magic number as well, as `<DataTable :rows:"10">`. There's `:rows-per-page-options="[10, 20, 50]"` but otherwise that's it afaik.